src/PaymentSuite/RedsysBundle/phpunit.xml.dist

```diff
@@ -27,8 +27,5 @@
             </exclude>
         </whitelist>
     </filter>
-    <php>
```

Comment:
@mickaelandrieu I added a dependency for https://github.com/elcodi/TestCommonBundle, which allows tests extending the `Elcodi\Bundle\TestCommonBundle\Functional\WebTestCase` to auto-discover the `KERNEL_DIR`.
Setting the `KERNEL_DIR` explicitly in the `phpunit.xml.dist` will force you to have a single symfony application, while we have several packages (one for each payment bundle) that can have their own `AppKernel` class.